samples/subsys/usb/uac2_explicit_feedback/prj.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_I2S=y
+
+#USB related configs
+CONFIG_USB_DEVICE_STACK_NEXT=y
+CONFIG_USBD_AUDIO2_CLASS=y
+CONFIG_SAMPLE_USBD_PID=0x000E
+CONFIG_SAMPLE_USBD_PRODUCT="UAC2 explicit feedback sample"
+CONFIG_NET_BUF=y
```

Comment:
shouldn't the USB subsystem enable this?

Reply:
Yes, it does. Removed.